tests/test_markdown_links.py

```
@@ -0,0 +1,708 @@
+import pytest
+import unittest.mock as mock
+from pathlib import Path
+import tempfile
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)